src/iotEdgeExplorer.ts

```diff
@@ -102,6 +114,23 @@ export class IoTEdgeExplorer extends BaseExplorer {
         }
     }
 
+    private async isValidDeploymentJson(json: string): Promise<boolean> {
+        try {
+            const schema = (await axios.get(Constants.DeploymentJsonSchemaUrl)).data;
+            const ajv = new Ajv({ allErrors: true });
+            const valid = ajv.validate(schema, JSON.parse(json));

```

Comment:
JSON.parse(json) may throw error, and we need catch this error to let user know that it is not a json string.

Reply:
So the validation includes two parts. 1. verify it is a valid json 2. Verify the schema is correct.

Reply:
Fixed